code/modules/shuttle/shuttle.dm

```diff
@@ -311,6 +311,8 @@
 			comp.connect_to_shuttle(src, dock, idnum)
 		for(var/obj/machinery/computer/camera_advanced/shuttle_docker/comp in place)
 			comp.connect_to_shuttle(src, dock, idnum)
+		for(var/obj/machinery/status_display/shuttle/sd in place)
+			sd.connect_to_shuttle(src, dock, idnum)
```

Comment:
We're getting enough of these we may want to consider in the future making this an atom proc or tying it into init

Reply:
Agreed, saving it for a future PR though